src/utilities/layout-engine/horizontal-tree.js

```diff
@@ -83,7 +83,9 @@ class HorizontalTree extends SnapToGrid {
 
     if (nodeLocationOverrides) {
       for (const gNodeId in nodeLocationOverrides) {
-        if (nodeLocationOverrides.hasOwnProperty(gNodeId)) {
+        if (
+          Object.prototype.hasOwnProperty.call(nodeLocationOverrides, gNodeId)
```

Comment:
Why did this need to change?

Reply:
The new version of eslint doesn't accept the usage of prototype builtins. 
(see the following rule: [https://eslint.org/docs/rules/no-prototype-builtins](https://eslint.org/docs/rules/no-prototype-builtins) )